components/feature/top-sites/src/main/java/mozilla/components/feature/top/sites/TopSitesConfig.kt

```diff
@@ -29,8 +29,10 @@ data class TopSitesConfig(
  * @property showProviderTopSites Whether or not to display the top sites from the provider.
  * @property maxThreshold Only fetch the top sites from the provider if the number of top sites are
  * below the maximum threshold.
+ * @property topSitesProviderFilter Function used to filter the sponsored shortcuts.
```

Comment:
```suggestion
 * @property topSitesProviderFilter Optional function used to filter the top sites from the provider.
```

Reply:
Got this comment from another team member. Let's rename this to `providerFilter` to avoid repetitive naming here as well,